scripts/install_plugin_modules.js

```diff
@@ -130,8 +130,14 @@ async function assertPackage (name, version, dependency, external) {
   }
 
   if (!external) {
-    pkg.workspaces = {
-      nohoist: ['**/**']
+    if (name === 'aerospike') {
```

Comment:
This exception just for one package looks interesting. What's the story here?

Reply:
`Command: npm install @mapbox/node-pre-gyp
Arguments: 
Directory: /home/runner/work/dd-trace-js/dd-trace-js/versions/aerospike/node_modules/aerospike
Output:
TypeError: workspaces config expects an Array`

Running yarn services, was giving me this error^. Tried a bunch of things to get it to work, & this suggestion made it work:

https://stackoverflow.com/questions/71747375/migrating-to-yarn-3-workspaces-nohoist

Reply:
What's the harm of always providing both options?

Reply:
then aerospike specifically won't build because it has this set:

pkg.workspaces = {
      nohoist: ['**/**']
  }

Reply:
Aren't the 2 options equivalent? I'm not sure I understand the explanation in the above link.

Reply:
Technically they are, but for some reason aerospike can't build with the yarn services command if this is set pkg.workspaces = {
nohoist: ['/']
}

Reply:
Ok, we can always figure out this one later.